server/routes/products.js

```diff
@@ -0,0 +1,142 @@
+const productRouter = require("express").Router();
```

Comment:
May want to check if someone is an admin before allowing them to create products